Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN conda install -c conda-forge \
         tqdm && \
         conda clean -ya && \
         pip install kaggle tqdm && \
-        conda install -c rapidsai -c nvidia -c conda-forge -c defaults cudf=0.15.0 dask-cuda rmm librmm rapids-xgboost cuml=0.15
+        conda install -c rapidsai -c nvidia -c conda-forge -c defaults cudf=0.15.0 dask-cuda rmm librmm rapids-xgboost cuml=0.15 xgboost=1.1.1
```

Comment:
Why do we use XGBoost 1.1.1 here?

Reply:
I meant to remove that, sorry. I just committed another change.